engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/content-host-details-info.controller.js

```diff
@@ -131,7 +131,7 @@ angular.module('Bastion.content-hosts').controller('ContentHostDetailsInfoContro
         };
 
         $scope.convertMemToGB = function (memoryValue) {
-            return (memoryValue / 1024000).toFixed(2);
+            return (memoryValue/1000000).toFixed(2);
```

Comment:
https://github.com/candlepin/candlepin/blob/master/server/src/main/resources/rules/rules.js#L543

These facts are important to display properly to users as they are related to how subscriptions are consumed. Switching unit measurement probably not a good idea. :)